irnl_rdt_correction/input_options.py

```diff
@@ -0,0 +1,219 @@
+            'update_optics': True,
+            'iterations': 1,
+            'ignore_corrector_settings': False,
+            'rdts2': None,
+            'ignore_missing_columns': False,
+            'output': None,
+            }
+
+DEFAULT_RDTS = {
+    'lhc': ('F0003', 'F0003*',  # correct a3 errors with F0003
+            'F1002', 'F1002*',  # correct b3 errors with F1002
+            'F1003', 'F3001',  # correct a4 errors with F1003 and F3001
+            'F4000', 'F0004',  # correct b4 errors with F4000 and F0004
+            'F6000', 'F0006',  # correct b6 errors with F6000 and F0006
+            ),
+    'hllhc': ('F0003', 'F0003*',  # correct a3 errors with F0003
+              'F1002', 'F1002*',  # correct b3 errors with F1002
+              'F1003', 'F3001',  # correct a4 errors with F1003 and F3001
+              'F0004', 'F4000',  # correct b4 errors with F0004 and F4000
+              'F0005', 'F0005*',  # correct a5 errors with F0005
+              'F5000', 'F5000*',  # correct b5 errors with F5000
+              'F5001', 'F1005',  # correct a6 errors with F5001 and F1005
+              'F6000', 'F0006',  # correct b6 errors with F6000 and F0006
+              ),
+}
+
+
+# Parser -----------------------------------------------------------------------
+
+def get_parser() -> argparse.ArgumentParser:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--twiss",
+        dest="twiss",
+        nargs="+",
+        help="Path(s) to twiss file(s). Defines which elements to correct for!",
+        required=True,
+    )
+    parser.add_argument(
+        "--errors",
+        dest="errors",
+        nargs="+",
+        help="Path(s) to error file(s).",
+        required=True,
+    )
+    parser.add_argument(
+        "--beams",
+        dest="beams",
+        type=int,
+        nargs="+",
+        help="Which beam the files come from (1, 2 or 4)",
+        required=True,
+    )
+    parser.add_argument(
+        "--output",
+        dest="output",
+        help=("Path to write command and tfs_df file. "
+              f"Extension (if given) is ignored and replaced with {EXT_TFS} and {EXT_MADX} "
+              "for the Dataframe and the command file respectively."
+              ),
+    )
+    parser.add_argument(
+        "--rdts",
+        dest="rdts",
+        nargs="+",
+        help=("RDTs to correct."
+              " Format: 'Fjklm'; or 'Fjklm*'"
+              " to correct for this RDT in Beam 2 using"
+              " beta-symmetry (jk <-> lm)."),
+    )
+    parser.add_argument(
+        "--rdts2",
+        dest="rdts2",
+        nargs="+",
+        help=("RDTs to correct for second beam/file, if different from first."
+              " Same format rules as for `rdts`."),
+    )
+    parser.add_argument(
+        "--accel",
+        dest="accel",
+        type=str.lower,
+        choices=list(DEFAULT_RDTS.keys()),
+        default=DEFAULTS['accel'],
+        help="Which accelerator we have.",
```

Comment:
which we use .. what's the implications, is it energy/brho?

Reply:
nothing. but the options are only 'lhc' or 'hllhc' because they have different corrector naming